.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-	"presets": ["es2015", "react"],
-	"plugins": ["transform-object-rest-spread", "transform-es2015-modules-commonjs"]
+	"presets": ["es2015", "stage-0", "react"],
```

Comment:
I see, what about add 'babel-plugin-transform-object-assign' or 'babel-plugin-transform-runtime', that will work too :)